lib/Notes/NotesAccordion/NotesAccordion.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+} from 'react-intl';
+import {
+  get,
+} from 'lodash';
+
+import {
+  Accordion,
+  Headline,
+  Badge,
+  Button,
+} from '@folio/stripes-components';
+
+import NotesList from './components';
+import styles from './NotesAccordion.css';
+
+export const noteShape = PropTypes.shape({
+  id: PropTypes.string,
+  lastSavedDate: PropTypes.instanceOf(Date),
+  lastSavedUserFullName: PropTypes.string,
+  title: PropTypes.string,
+});
+
+export class NotesAccordion extends Component {
+  static propTypes = {
+    notes: PropTypes.arrayOf(noteShape),
+    onCreate: PropTypes.func.isRequired,
+    onNoteClick: PropTypes.func.isRequired,
+    open: PropTypes.bool,
+  }
+
+  renderHeader = () => {
+    return (
+      <Headline
+        data-test-notes-accordion-header
+        size="large"
+        tag="h3"
+      >
+        <FormattedMessage id="stripes-smart-components.notes" />
+      </Headline>
+    );
+  }
+
+  renderHeaderButtons() {
+    const {
+      onCreate,
+    } = this.props;
+
+    return (
+      <Fragment>
+        <Button
+          data-test-notes-accordion-add-button
+        >
+          <FormattedMessage id="stripes-smart-components.add" />
+        </Button>
+        <Button
+          data-test-notes-accordion-new-button
+          buttonClass={styles['new-button']}
+          onClick={onCreate}
+        >
+          <FormattedMessage id="stripes-smart-components.new" />
+        </Button>
+      </Fragment>
+    );
+  }
+
+  getNotesQuantity() {
+    return get(this.props.notes, 'length');
```

Comment:
I think if we make `notes` prop default to `[]`, we'll be able to write just `this.props.notes.length` here and `lodash.get` won't be necessary. This way `getNotesQuantity` will return `0` in case when there is no notes, which is more sensible than `undefined`.
Also I guess `getNotesQuantity` will be unnecessary too. You will be able to just use `this.props.notes.length` inside of the `renderQuantityIndicator`:
```js
renderQuantityIndicator() {
    return (
      <Badge>
        <span data-test-notes-accordion-quantity-indicator>
          {this.props.notes.length}
        </span>
      </Badge>
    );
  }
```